CMakeLists.txt

```diff
@@ -63,19 +66,34 @@ set(PUBLIC_HEADERS
 
 add_library(cppcodec OBJECT ${PUBLIC_HEADERS}) # unnecessary for building, but makes headers show up in IDEs
 set_target_properties(cppcodec PROPERTIES LINKER_LANGUAGE CXX)
-add_subdirectory(tool)
-add_subdirectory(example)
 
-if (BUILD_TESTING)
+target_include_directories(cppcodec PUBLIC
```

Comment:
Style nitpick: a single  `target_include_directories()` can contain both `PUBLIC` and `PRIVATE` sections, so I think it's cleaner to move the `PUBLIC` into the next line (so a hypothetical `PRIVATE` wouldn't look out of place, not that it matters much here).

Reply:
Yes